code/caesar_cipher/model/caesar_cipher.py

```diff
@@ -0,0 +1,33 @@
+import string
+
+
+class CaesarCipher(object):
+    def __init__(self, offset):
+        self.alphabetVolume = string.ascii_lowercase.__len__()
+        self.alphabet = {x: string.ascii_lowercase[x] for x in range(self.alphabetVolume)}
+        self.letterMap = {string.ascii_lowercase[x]: x for x in range(self.alphabetVolume)}
+        self.offset = offset % self.alphabetVolume
```

Comment:
What happens if we pass an array as a shift value? Please add tests for incorrect data